common/workunit/workunit.hpp

```diff
@@ -1659,7 +1659,7 @@ extern WORKUNIT_API const char * getWorkunitActionStr(WUAction action);
 extern WORKUNIT_API WUAction getWorkunitAction(const char * actionStr);
 
 extern WORKUNIT_API void addTimeStamp(IWorkUnit * wu, StatisticScopeType scopeType, const char * scope, StatisticKind kind, unsigned wfid=0);
-extern WORKUNIT_API double calculateThorCost(__int64 timeNs, unsigned clusterWidth);
+extern WORKUNIT_API double calculateThorCost(double elapsedTime, unsigned clusterWidth);
```

Comment:
I think 'elapsedTime' units are seconds? But would be clearer if elapsedTimeSecs